### DISCUSSION TITLE
Cursor out of text input

### DISCUSSION THREAD

**Discussion by BruceMustache** [8VIM/8VIM#125] [Ideas]
Bug: moving cursor gets it out of text input

**Comment by Glitchy-Tozier** [comment]
What does that mean?

**Comment by BruceMustache** [comment]
For example, open any app and slide up from center
You'll see the cursor gets out of input section 
```
  A   B   C
 ___________
```

Let's say underline is the input section
And the application you're on have 3 buttons
A, B, C
When you slide from center to up, moving cursor up
The focus goes to the button A instead of stay on underline
Similar behavior happens if you use arrow keys in a physical keyboard or emulate it with a android emulator

**Comment by Glitchy-Tozier** [comment]
I think I understand. To me personally, this sounds fun though and i wouldn't mind keeping it.
But let's see what other people think.

**Comment by m-rey** [comment]
I like the functionality a lot, as it allows using the keyboard for even more things.
Rather than removing the behaviour, this functionality should be documented in the readme. See #109
